src/bundle/Resources/public/scss/_user-menu.scss

```diff
@@ -52,7 +52,7 @@
         &--hidden {
             height: 0;
             overflow: hidden;
-            z-index: initial;
+            z-index: 1050;
```

Comment:
You can remove this line. It will be inherited from https://github.com/ezsystems/ezplatform-admin-ui/pull/596/files#diff-e23bb5b2bd398427b2df11a794cc7729R47

Reply:
Yes, menu it's still clickable. Adding video http://recordit.co/RNPrPOv3k2